documentation/tutorials/buildlinux.rst

```diff
@@ -81,7 +81,11 @@ And it can be even simpler by just running::
 
   kw b
 
+.. note:
+    kw will also run `make olddefconfig` for you before building, so you can
```

Comment:
Maybe:
kw runs `...` before start building...

Also, you can drop the other part that says "so you..."